documentation/C++style.md

```diff
@@ -15,7 +17,15 @@ is clear on usage, follow it.
 * Reasonable exceptions to these guidelines can be made.
 * Be aware of some workarounds for known issues in older C++ compilers that should
   still be able to compile f18. They are listed at the end of this document.
+
 ## In particular:
+
+Use serial commas in comments, error messages, and documentation.
+
+### Error messages
+1. Fortran keywords should appear in upper case.
+1. Names from the program appear in single quotes.
```

Comment:
Omit the period if a message comprises a single sentence (complete or not).

Reply:
Messages should be a single sentence with very few exceptions. The only multi-sentence message I could find was:
`TiesAwayFromZero rounding mode is not available not available when folding constants with host runtime. Using TiesToEven instead.`
Even in that case I think a colon would be better.

And I see there's a typo with "not available" duplicated.

Reply:
My mistake, I will fix this.
Question regarding this message and and one that I am adding in #749. Would it be better to put the second part, that states how f18 handles the issue, as a note attached to the main message ?

Similar message in #749 is:
```
"Argument type does not match specific intrinsic '%s' requirements,"
" using '%s' generic instead and converting the result to %s if needed"_en_US,
```

Reply:
I think it's good having it all in one message. I suggest:
`TiesAwayFromZero rounding mode is not available when folding constants with host runtime; using TiesToEven instead`

The other one is probably better with a semi-colon instead of the comma as well.

Reply:
Thanks for the feedback. Both messages are fixed in commit https://github.com/flang-compiler/f18/pull/749/commits/621ded353f6179045e8fc4c18ef13a4aa05293b9 in PR #749.